src/app/(proper_react)/redesign/Shell.tsx

```diff
@@ -12,87 +12,92 @@ import { MobileShell } from "./MobileShell";
 import Link from "next/link";
 import { PageLink } from "./PageLink";
 import { ExtendedReactLocalization } from "../../hooks/l10n";
+import { GaScript } from "./GaScript";
 
 export type Props = {
   l10n: ExtendedReactLocalization;
   session: Session;
   children: ReactNode;
+  nonce: string;
 };
 
 export const Shell = (props: Props) => {
   const l10n = props.l10n;
 
   return (
-    <MobileShell session={props.session}>
-      <div className={styles.wrapper}>
-        <nav
-          className={styles.mainMenu}
-          aria-label={l10n.getString("main-nav-label")}
-        >
-          <Link href="/" className={styles.homeLink}>
-            <Image
-              src={monitorLogo}
-              alt={l10n.getString("main-nav-link-home-label")}
-              width={170}
-            />
-          </Link>
-          <ul>
-            {/* Note: If you add elements here, also add them to <MobileShell>'s navigation */}
-            <li>
-              <PageLink
-                href="/redesign/user/dashboard"
-                activeClassName={styles.isActive}
-              >
-                {l10n.getString("main-nav-link-dashboard-label")}
-              </PageLink>
-            </li>
-            <li>
-              <a
-                href="https://support.mozilla.org/kb/firefox-monitor-faq"
-                title={l10n.getString("main-nav-link-faq-tooltip")}
-              >
-                {l10n.getString("main-nav-link-faq-label")}
-              </a>
-            </li>
-          </ul>
-        </nav>
-        <div className={styles.content}>
-          <div className={styles.page}>{props.children}</div>
-          <footer className={styles.footer}>
-            <a
-              href="https://www.mozilla.org"
-              className={styles.mozillaLink}
-              target="_blank"
-            >
+    <>
+      <GaScript nonce={props.nonce} />
```

Comment:
I'm assuming the only change in the huge diff below is indentation, right?

Reply:
Yes, that’s right!